frontend/src/features/verifiable-fields/VerifiableFieldProvider.tsx

```diff
@@ -22,6 +21,7 @@ import { VerifiableFieldContext } from './VerifiableFieldContext'
 
 export interface VerifiableFieldProps extends BaseFieldProps {
   schema: VerifiableFieldSchema<FormFieldWithId<VerifiableFieldBase>>
```

Comment:
I think it would be great to add more documentation throughout our React code, but particularly for usage of react context. it can be pretty difficult to follow when reading it for the first time. maybe we can at least explain each variable stored in the context?

Reply:
Done. Added docs to VerifiableFieldProvider.